lib/obj.js

```diff
@@ -656,6 +656,17 @@ function saveMetadata(req, res, next) {
                     res.header('Access-Control-Allow-Origin',
                                req.headers['origin']);
                 }
+                /*
+                 * The following headers will have already been set to the
```

Comment:
@bahamas10 Perhaps we could update this comment with some specifics:
- restify/node-restify#693 makes restify's `response.header(name, value)`
  atypically accumulate multiple values
- Etag and Last-Modified are also set earlier by the `getMetadata` handler
  in common.js

Reply:
will do